action.yml

```diff
@@ -1,18 +1,24 @@
-name: 'Code Dx Analysis'
-description: 'Analyze your source code and binaries with Code Dx'
+name: 'SRM Analysis'
+description: 'Analyze your source code and binaries with SRM'
 inputs:
   # main config options
   server-url:
-    description: 'the URL for the Code Dx server (typically ends with `/codedx`)'
+    description: 'the URL for the SRM server (typically ends with `/codedx`)'
     required: true
   api-key:
-    description: 'an API key or Personal Access Token (PAT) to use when connecting to Code Dx'
+    description: 'an API key or Personal Access Token (PAT) to use when connecting to SRM'
     required: true
   project-id:
-    description: 'the ID of a project (an integer) created in Code Dx'
+    description: 'the ID of a project (an integer) created in SRM'
     required: true
+  base-branch-name:
+    description: 'the parent branch name of a project created in SRM'
+    required: false
+  target-branch-name:
+    description: 'the target branch name of a project created in SRM'
+    required: false
```

Comment:
Should mention that it automatically creates the branch if it doesn't exist yet in the project, and the new branch will be created from `base-branch-name`